src/osd/ECBackend.cc

```diff
@@ -2426,8 +2426,6 @@ int ECBackend::be_deep_scrub(
 {
   dout(10) << __func__ << " " << poid << " pos " << pos << dendl;
   int r;
-  bool skip_data_digest = store->has_builtin_csum() &&
-    g_conf->osd_skip_data_digest;
```

Comment:
Not currently at a computer and I missed part of the story here, but I take it from his construction that osd_skip_data_digest defaults to true? I’m a bit confused about this setting’s use.